Library/Homebrew/requirements/cmake_requirement.rb

```diff
@@ -0,0 +1,32 @@
+require "requirement"
+
+class CmakeRequirement < Requirement
+  fatal true
+  default_formula "cmake"
+
+  def initialize(tags)
+    @version = tags.shift if /\d+\.*\d*/ === tags.first
+    raise "Specify a version for CmakeRequirement" unless @version
+    super
+  end
+
+  satisfy :build_env => false do
+    next unless which "cmake"
+    cmake_version = Utils.popen_read("cmake", "--version").split(" ").last
+    Version.new(cmake_version) >= Version.new(@version)
+  end
+
+  env do
+    ENV.prepend_path "PATH", which("cmake").dirname
```

Comment:
Not 100% sure, but if my cmake binary sits in say /usr/bin/, this will pull that into the path possibly messing with the isolated build environment or not?